snippets/python/flamesGame.md

```diff
@@ -0,0 +1,80 @@
+# This is a fun game. Come up with a program that displays the relationship between two persons.
```

Comment:
Edit this title, remove "come up". Also rewrite the heading to convey the same thing